wgpu/src/lib.rs

```diff
@@ -734,6 +734,7 @@ impl Drop for ShaderModule {
 }
 
 /// Source of a shader module.
+#[derive(Clone)]
```

Comment:
I don't think you ever want the `Clone` derive here.
For `Cow` type, it just clones the contents instead of re-borrowing them - https://doc.rust-lang.org/stable/src/alloc/borrow.rs.html#194-211

If you want `Clone`, I suggest just implementing it manually, using `Cow::Borrowed`